packages/@aws-cdk/aws-lambda-event-sources/README.md

```diff
@@ -67,6 +67,7 @@ const queue = new sqs.Queue(this, 'MyQueue', {
 
 lambda.addEventSource(new SqsEventSource(queue, {
   batchSize: 10, // default
+  maxBatchingWindow: Duration.minutes(5),
```

Comment:
Can you add a couple of sentences briefly describing batching (with a link for further details) above this code snippet?